endpoints/schema.yml

```diff
@@ -6,88 +6,77 @@ paths:
           in: query
           description: tags to filter by
           required: false
-          type: array
-          items:
-            type: string
-          collectionFormat: csv
+          type: string
```

Comment:
Why the change?

Reply:
two reasons:
1. our swagger ui doesn't support all properties, i think keep the basic parameters is enough.
2. even us don't use properties like `collectionFormat`, i was trying to not give users too many properties to learn. just keep the simple or required ones.

we can add more properties though, but it will not be a simple START example for users.

Reply:
ok